api/package.json

```diff
@@ -5,16 +5,18 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
```

Comment:
you can directly run `jest`

Reply:
When I directly run `jest` it's complaining of the import statement.

Reply:
Google how to solve it.